docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+Creating BREEDS subpopulation shift benchmarks
+===============================================
+
+In this document, we will discuss how to create BREEDS datasets, used in 
+the `code <https://github.com/MadryLab/BREEDS_benchmarks>`_ `release
+<https://github.com/MadryLab/BREEDS_benchmarks>`_ based on our paper [STM20]_. 
+These datasets can be used to study model robustness to subpopulation 
+shift---a specific form of distribution shift wherein the subpopulations of data 
+(e.g., animal breeds) present in the training set (source domain) are 
+entirely different from the test set (target domain). 
+You can create BREEDS datasets from any existing dataset that has 
+an explicit class hierarchy describing the semantic relationship 
+between pairs of classes (e.g., ImageNet, OpenImages). 
+In this walkthrough, we will do so using ImageNet and the corresponding
+class hierarchy from [STM20]_ (modification of WordNet to be better-suited
+for the object recognition task).
+
+At a high-level, the procedure to construct BREEDS datasets is as follows:
+
+1. Use the dataset's class hierarchy to define `superclasses` by grouping
+   together semantically similar dataset classes (e.g., define the `dog`` superclass 
+   by aggregating all the dog breed classes). 
+2. Design the classification task to be between different superclasses, so that
+   we have easy access to the individual subpopulations (i.e., the original 
+   dataset classes). Then, for each superclass, we will partition the subpopulations 
+   into the source (used for model training) and target domain without overlap
+   (used for evaluation). 
+
+
+.. raw:: html
+
+   <i class="fa fa-play"></i> &nbsp;&nbsp; <a
+   href="https://github.com/MadryLab/BREEDS-Benchmarks/blob/master/Constructing%20BREEDS%20datasets.ipynb">Download
+   a Jupyter notebook</a> containing all the code from this walkthrough! <br />
+   <br />
+
+Requirements/Setup
```

Comment:
Would be nice to have a setup_breeds(root=SOME_DIR) function (noted this below too) that downloads all of this stuff and puts it in root, that way they don't have to clone the other repo.

Reply:
Done.